Doc/library/asyncio-eventloop.rst

```diff
@@ -361,7 +361,7 @@ Creating Futures and Tasks
 
    .. versionadded:: 3.5.2
 
-.. method:: loop.create_task(coro, *, name=None, context=None, eager_start=None)
+.. method:: loop.create_task(coro, *, name=None, context=None, eager_start=None, **kwargs)
```

Comment:
IIUC we ended up with **just** ``**kwargs``.

Reply:
I think it should be just `**kwargs` for 3.14 no need to manually list `name` and `context`.

Reply:
Well, we do need to give users guidance on what arguments are accepted by create_task in a vanilla set-up (no 3rd party loop, no task factory). We could link to `Task` for that (DRY), but since we really don't want users to call `Task` explicitly I think we need to show the vanilla set of arguments in create_task. Also, we expect all flavors to support `name` and probably also `context`, and we want to document (advertise) the per-task override of `eager_start` here as well.

Reply:
In this PR I took the example from the kwargs to subprocess.run which has a similar relationship with subprocess.Popen.

I think repeating the docs for all these keyword arguments is not so bad when we have just 3. Perhaps reconsidering if more are added.

> (EDIT: Since it is supported by the default Task implementation.)

Note that you must use the default Task implementation (you can wrap it or subclass it but not swap it) otherwise `asyncio.current_task`/`all_tasks` do not work correctly

Reply:
Ok.